src/views/HomeNew/components/ModelsBySolutions/index.test.tsx

```diff
@@ -89,6 +94,7 @@ const mocks = [
             modelPlan: {
               id: '4fc87324-dbb0-4867-8e4d-5a20a76c8ae2',
               modelName: 'Plan with Basics',
+              modelBySolutionStatus: ModelBySolutionStatus.ENDED,
```

Comment:
Should we add a case for `ModelBySolutionStatus.OTHER` here?

Reply:
Good call, I'll add it.

Reply:
Added in [50aa733](https://github.com/CMSgov/mint-app/pull/1233/commits/50aa733351ba687742b771e9fac551ed3c085ae8)